src/curated-corpus/components/ExistingProspectCard/ExistingProspectCard.tsx

```diff
@@ -85,30 +101,41 @@ export const ExistingProspectCard: React.FC<ExistingProspectCardProps> = (
           >
             <span>{item.publisher}</span>
           </Typography>
-          <Box py={1}>
-            <Chip
-              variant="outlined"
-              color="primary"
-              label={getDisplayTopic(item.topic)}
-              icon={<LabelOutlinedIcon />}
-            />{' '}
-            {item.isSyndicated && (
+          <Grid container spacing={1}>
+            <Grid item>
               <Chip
                 variant="outlined"
                 color="primary"
-                label="Syndicated"
-                icon={<CheckCircleOutlineIcon />}
+                label={getDisplayTopic(item.topic)}
+                icon={<LabelOutlinedIcon />}
+              />
+            </Grid>
+            <Grid item>
+              {item.isSyndicated && (
+                <Chip
+                  variant="outlined"
+                  color="primary"
+                  label="Syndicated"
+                  icon={<CheckCircleOutlineIcon />}
+                />
+              )}
+            </Grid>
+            <Grid item>
+              <Chip
+                variant="default"
+                color="secondary"
+                label="Already in corpus"
+                icon={<CheckIcon />}
               />
-            )}{' '}
-            <Chip
-              variant="default"
-              color="secondary"
-              label="Already in corpus"
-              icon={<CheckIcon />}
-            />
-          </Box>
+            </Grid>
+          </Grid>
```

Comment:
Replaced the above jsx from `<Box>` to `<Grid>` to get achieve some spacing. Although, a lot more lines to achieve something little. I could probably use a CSS class and make the styling work with the `<Box>`.

Okay with reverting it if you think it's a bit verbose/lengthy?

Reply:
I think it's all right as long as we make corresponding changes to the Prospect card component which also has these pill-style tags.